client/my-sites/importer/importer-logo.jsx

```diff
@@ -12,9 +12,10 @@ import SocialLogo from 'social-logos';
  */
 import GoDaddyGoCentralLogo from './logos/godaddy-gocentral';
 import WixLogo from './logos/wix';
+import MediumLogo from './logos/medium';
 
 const ImporterLogo = ( { icon } ) => {
-	if ( includes( [ 'wordpress', 'medium', 'blogger-alt', 'squarespace' ], icon ) ) {
+	if ( includes( [ 'wordpress', 'blogger-alt', 'squarespace' ], icon ) ) {
```

Comment:
Noting that it would be nice to use constants for the icon type, but I don't think we should do that as part of this change.